### PR TITLE
Sirius testnet reset documentation

### LINKED ISSUE #499: Documentation updates for testnet reset
There will be documentation updates required for the testnet reset.

**Sub-tasks**
- [ ] Release notes (and what to do with the existing Testnet release notes)
- [ ] Downloads page, in particular genesis hash 
- [ ] Maybe other documentation about node migration?

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Documentation for what users need to do for testnet reset.

## Changes

Added topics in Resources folder for nodes, wallets, client. Started update of testnet downloads page. Added release note.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

Closes #499

### CODE REVIEW

File: source/mainnet/net/resources/sirius-testnet-reset-nodes.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+To update the Windows node you need to delete the existing testnet database and
+install and run the new node. The steps are:
+
+1. Stop the existing testnet node following :ref:`instructions
+   <install-upgrade-node-windows>` on how to upgrade a node.
+2. Navigate to the testnet node's data directory. This was selected during node
+   installation. By default it will be ``<CommonAppDataFolder>\Concordium\Node
+   Runner\testnet\data`` where ``<CommonAppDataFolder>`` is usually
+   ``C:\ProgramData``.
+3. Delete the directory named ``database-v4``.
```
Comment (mh-concordium) [inline source/mainnet/net/resources/sirius-testnet-reset-nodes.rst:21]: This didn't work. I had to delete everything in data folder before it actually worked. I am not sure why I was getting an error message that Genesis is wrong. Maybe installer didn't overwrite old Genesis.dat.

Comment (abizjak) [reply source/mainnet/net/resources/sirius-testnet-reset-nodes.rst]: I updated the instructions to delete the testnet\data folder.

File: source/mainnet/net/resources/sirius-testnet-reset-nodes.rst [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+4. `Download the Testnet macOS installer package <https://distribution.concordium.software/macos/signed/concordium-node-4.1.0-0.pkg>`_
+5. Install the new node version and start it.
+
+Ubuntu
+======
+
+To update Ubuntu, you need to download and intall the new version.
+
+`Download the Testnet Debian package <https://distribution.testnet.concordium.com/deb/concordium-testnet-node_4.1.0_amd64.deb>`_
```
Comment (EmilLa1) [inline source/mainnet/net/resources/sirius-testnet-reset-nodes.rst:44]: I believe this should be the 4.1.1 ?

And likewise the sha256 sum should be corrected.

Comment (EmilLa1) [inline source/mainnet/net/resources/sirius-testnet-reset-nodes.rst:44]:
```suggestion
`Download the Testnet Debian package <https://distribution.testnet.concordium.com/deb/concordium-testnet-node_4.1.1_amd64.deb>`_
```

File: source/mainnet/net/resources/sirius-testnet-reset-nodes.rst [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+      - SHA256 checksum of the download: ``ad418857eb27ca17b0747198f91a1bee89728ba91d0d14965fa614af23d9d0f6``
```
Comment (EmilLa1) [inline source/mainnet/net/resources/sirius-testnet-reset-nodes.rst:46]:
```suggestion
      - SHA256 checksum of the download: ``35c668769576e32d94f53b3c167b16ad7c74425737932f9b9f2837ea35fe3641``
```

**Review by EmilLa1 — CHANGES_REQUESTED** [review]

File: source/mainnet/net/resources/sirius-testnet-reset-nodes.rst (hunk 2 above)
Comment (dg-concordium) [reply source/mainnet/net/resources/sirius-testnet-reset-nodes.rst]: I just updated that page.

Comment (dg-concordium) [reply source/mainnet/net/resources/sirius-testnet-reset-nodes.rst]: Also updated that.

File: source/mainnet/net/resources/sirius-testnet-reset-nodes.rst (hunk 3 above)
Comment (dg-concordium) [reply source/mainnet/net/resources/sirius-testnet-reset-nodes.rst]: And I updated that.

File: source/mainnet/net/resources/sirius-testnet-reset-nodes.rst (hunk 2 above)
Comment (EmilLa1) [reply source/mainnet/net/resources/sirius-testnet-reset-nodes.rst]: Great!

File: source/mainnet/net/resources/sirius-testnet-reset-nodes.rst (hunk 3 above)
Comment (EmilLa1) [reply source/mainnet/net/resources/sirius-testnet-reset-nodes.rst]: Great!

**Review by EmilLa1 — APPROVED** [review]
Can confirm that the information related to Ubuntu is OK.

**Review by abizjak — APPROVED** [review]

**Pull request merged**